client/src/components/OnboardUser.js

```diff
@@ -0,0 +1,233 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import NavBar from './NavBar';
+import { ReactComponent as Logo } from '../img/Logo_CORONATRACKER_Logo.svg';
+import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.svg';
+import profileImg from '../img/profile.png';
+
+const useStyles = makeStyles(theme => ({
+  //the styles goes here as an object
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    margin: '2em 0 2em 0',
+  },
+  button: {
+    color: 'white',
+    backgroundColor: 'red',
+    fontWeight: 'bold',
+    alignSelf: 'center',
+    height: '4vh',
+    borderRadius: '5px',
+    margin: '0 0 0 0',
+    maxWidth: '300px',
+    minWidth: '42%',
+  },
+  header: {
+    width: 325,
+    display: `inline-block`,
+    marginBottom: '.5em',
+    fontSize: '.5em',
+  },
+  profile: {
+    width: '100px',
+    height: '140px',
+    marginBottom: '.5em',
+    fontSize: '1em',
+  },
+  image: {
+    width: '100%',
+  },
+  inputArea: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: '.5vh',
+  },
+  inputField: {
+    width: '48%',
+    textAlign: 'center',
+    maxWidth: '180px',
+    color: 'white',
+    backgroundColor: 'red',
+    height: '4vh',
+    justifyContent: 'space-between',
+    border: 'none',
+    borderRadius: '5px',
+    '&::placeholder': {
+      color: 'white',
+    },
+
+    '&:-ms-input-placeholder': {
+      color: 'white',
+    },
+
+    '&::-ms-input-placeholder': {
+      color: 'white',
+    },
+  },
+  inputFieldLabel: {
+    textAlign: 'right',
+    width: '48%',
+  },
+  location: {
+    border: 'none',
+    marginBottom: '10px',
+    textAlign: 'center',
+    color: 'white',
+    backgroundColor: 'red',
+    width: '33%',
+    maxWidth: '60px',
+    fontSize: '1em',
+    borderRadius: '5px',
+    fontFamily:
+      '-apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial,',
+    '&::placeholder': {
+      color: 'white',
+    },
+
+    '&:-ms-input-placeholder': {
+      color: 'white',
+    },
+
+    '&::-ms-input-placeholder': {
+      color: 'white',
+    },
+  },
+}));
+
+const blankForm = {
+  firstName: '',
+  age: '',
+  sex: '',
+  city: '',
+  state: '',
+  zip: '',
+};
+
+export default ({ postNewUser }) => {
+  const [formState, setFormState] = useState(blankForm);
+  const handleChange = e => {
+    e.preventDefault();
+    setFormState({
+      ...formState,
+      [e.target.name]: e.target.value,
+    });
+  };
```

Comment:
I think we should move this state logic to Redux, right? Since we'll potentially need this onboarding information elsewhere in the app (when the user opens the settings page, for example).

Reply:
You wouldn't want to use redux to hold the state of the fields in a form before it is submitted.  I noted in my PR that the flow would ideally be, this component receives a function that, when the user clicks "SAVE MY RESPONSES", posts the form data to a database, and then posts the response of the created data to redux.  You'd want to persist data to redux only after you know that it has been persisted in the database in order to keep your application in sync.  The useState you see here only holds the data while this form component is open.

Reply:
Ah okay, you're right. Sorry, I shouldn't be going through these PRs at work - I'm not reading carefully enough.

Reply:
No sweat.